src/server/src/healthcheck/topology-loaded.indicator.ts

```diff
@@ -22,9 +27,14 @@ export default class TopologyLoadedIndicator extends HealthIndicator {
         });
       })
     )) as [string, boolean][];
-    const layerStatus = Object.fromEntries(layerEntries);
-    const isHealthy = Object.values(layerStatus).every(status => status);
-    const result = this.getStatus(key, isHealthy, layerStatus);
+
+    const isHealthy = layerEntries.every(status => status);
```

Comment:
`isHealthy` was always returning `true`, since it was just checking the truthiness of the tuple itself. This should do the trick:

```suggestion
    const isHealthy = layerEntries.every(([_, status]) => status);
```